openedx/adg/lms/applications/models.py

```diff
@@ -354,6 +357,30 @@ def set_status(self, status):
         self.status = status
         self.save()
 
+    def send_application_status_update_email(self, message_for_applicant):
+        """
```

Comment:
shouldn't we write this function in helpers instead of models?

Reply:
Me and @harris-irfan had a discussion on this one and we were a bit confused in the 2 implementations and If we think like it is related to the application object then it makes some sense but in other case it is not performing some action over the object. Moving it to helpers as you were not part of discussion and you pointed it out.